File: projects/m1/029-decimal-to-binary/solutions/alessiacaporaso/decimal-to-binary.js
// convert number 
const decimalNumber = parseInt(prompt('Please, enter a number to convert'))
let q = decimalNumber;
let result = '';
let convertNumber = 0;
// divide the number until the number is less than 2 and display the binary representation of the number
while(q > 0){
  console.log(q > 0)
  convertNumber = q % 2;
  result += convertNumber.toString()
  q = Math.floor(q/2)
}
alert('The binary representation is: ' + result)

File: projects/m1/029-decimal-to-binary/solutions/MatteoFaienza/decimal-to-binary.py
user_number = int(input('Enter a integer number: '))

binari_number = ''

while user_number != 0 :

    if user_number % 2 == 0 :
        user_number //= 2
        binari_number += '0'
         

    else :
        user_number //= 2
        binari_number += '1' 

print('The binary number is',binari_number[::-1]) 